nav2_map_server/README.md

```diff
@@ -62,42 +140,112 @@ occupied_thresh: 0.65
+# Specify the parmeters file
+$ ros2 run nav2_map_server map_server_2d --ros-args --params-file <map_server_params_2d.yaml>
 ```
 
-There is also possibility of having multiple map server nodes in a single process, where the parameters file would separate the parameters by node name, like this:
+- 3D Server
+```shell
+# Provide the params filename accordinaly
+# Specify the parmeters file
+$ ros2 run nav2_map_server map_server_3d --ros-args --params-file <map_server_params_2d.yaml>
+```
 
+For 3D map server YAML consists of two main arguments file name as `image`, and view point as `origin` :
+```yaml
+image: testmap.pcd
+origin: [0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0]
 ```
-# combined_params.yaml
-map_server1:
+and the `parameters file` can be  defined as shown here:
+ 
+```yaml
+# map_server_params_3d.yaml
+map_server:
     ros__parameters:
-        yaml_filename: "some_map.yaml"
+        yaml_filename: "map_3d.yaml"
+```
 
-map_server2:
-    ros__parameters:
-        yaml_filename: "another_map.yaml"
+**Note:** * As any `LifecycleNode`, Map Server needs to be configured and activated via a call for the state change. This could be done for example by following code snippet:*
+
+```c++
+// map_client.cpp
+// Change state of node to configure
+#include "rclcpp/rclcpp.hpp"
+#include <memory>
+#include "nav2_util/lifecycle_service_client.hpp"
+
+using lifecycle_msgs::msg::Transition;
+
+static rclcpp::Node::SharedPtr node_ = 
+  rclcpp::Node::make_shared("map_client");
+  
+// Create the lifecycle client
+std::shared_ptr<nav2_util::LifecycleServiceClient> lifecycle_client_ = 
+  std::make_shared<nav2_util::LifecycleServiceClient>("map_server", node_);
+
+std::this_thread::sleep_for(std::chrono::seconds(5));
+const std::chrono::seconds timeout(5.0);
+
+// Configure and activate the node to launch the services
+lifecycle_client_->change_state(Transition::TRANSITION_CONFIGURE, timeout);
+lifecycle_client_->change_state(Transition::TRANSITION_ACTIVATE, timeout);
+
+// // // // // ///
+// Do something //
+// // // // // ///
+
+// Done with work, shut down the node
+lifecycle_client_->change_state(Transition::TRANSITION_DEACTIVATE);
+lifecycle_client_->change_state(Transition::TRANSITION_CLEANUP);
+lifecycle_client_->change_state(Transition::TRANSITION_UNCONFIGURED_SHUTDOWN);
 ```
 
-Then, one would invoke this process with the params file that contains the parameters for both nodes:
+> Build Instructions for the state changing client:
+
+```make - Makefile
+cmake_minimum_required(VERSION 3.5)
+
+project(map_server_client)
+
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(nav2_util REQUIRED)
+
+add_executable(map_client 
+  map_client.cpp)
 
+ament_target_dependencies(server_init
+  rclcpp
+  nav2_util)
+
+install(TARGETS server_init
+  RUNTIME DESTINATION lib/${PROJECT_NAME})
 ```
-$ process_with_multiple_map_servers __params:=combined_params.yaml
+
+> Run command for generated executable
```

Comment:
> Run command for generated executable

->

And command to run generated executable in parallel with `map_server_2d/map_server_3d`